xarray_sentinel/sentinel1.py

```diff
@@ -100,23 +101,17 @@ def open_orbit_dataset(product_path: str) -> xr.Dataset:
     shape = len(orbit)
 
     reference_system = orbit[0]["frame"]
-    data_vars: T.Dict[str, T.List[T.Any]] = {
-        "time": [],
-        "x": [],
-        "y": [],
-        "z": [],
-        "vx": [],
-        "vy": [],
-        "vz": [],
-    }
+    variables = ["time", "x", "y", "z", "vx", "vy", "vz"]
+    data_vars: T.Dict[str, T.List[T.Any]] = {var: ("time", []) for var in variables}  # type: ignore
+
     for k in range(shape):
-        data_vars["time"].append(orbit[k]["time"])
-        data_vars["x"].append(orbit[k]["position"]["x"])
-        data_vars["y"].append(orbit[k]["position"]["y"])
-        data_vars["z"].append(orbit[k]["position"]["z"])
-        data_vars["x"].append(orbit[k]["velocity"]["x"])
-        data_vars["y"].append(orbit[k]["velocity"]["y"])
-        data_vars["z"].append(orbit[k]["velocity"]["z"])
+        data_vars["time"][1].append(orbit[k]["time"])
```

Comment:
I `time` considered a coordinate? You add it to `data_vars` not `coords` and only test that it is a dimension, so I'm not sure.

Reply:
It's not necessary to add coordinates explicitly. If a variable and its dimension have the same name, then for xarray it is a coordinate. But probably would be better to add it explicitly.